README.md

```diff
@@ -1,126 +1,127 @@
 
 To get a local copy up and running, follow these steps.
 
-Prerequisites
+######Prerequisites
 In order to run this project you need: have a ssh key installed already
 
-Setup
-Clone this repository to your desired folder: cd "microverse-projects"
-git@github.com:arabone/Hello.git 
+######Setup
+Clone this repository to your desired folder: cd [("microverse-projects"
+git@github.com:arabone/Hello.git )]
 
-Install
+######Install
 Install this project with:ssh key file
 
-Usage
+######Usage
 To run the project, execute the following command:open with liveshare
 
-Run tests
+######Run tests
 To run tests, run the following command:start debugging
 
-Deployment
+######Deployment
 You can deploy this project using:chrome browser/netlify
 
 
 
-👥 Authors
-Mention all of the collaborators of this project.
+###👥 Authors
+>Mention all of the collaborators of this project.
 
 👤 Author1
 
-GitHub: https://github.com/arabone
-Twitter: @Mohamedarab8844
-LinkedIn: Mohamed Hassan
++GitHub: [(https://github.com/arabone)]
++Twitter: [(@Mohamedarab8844)]
++LinkedIn: [(Mohamed Hassan)]
 
 
 
-🔭 Future Features
-Describe 1 - 3 features you will add to the project.
+###🔭 Future Features
+>Describe 1 - 3 features you will add to the project.
 
- [databases]
- [client_server]
- [server_side]
+ *databases
+ *client_server
+ *server_side
 
 
-🤝 Contributing
+###🤝 Contributing
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the issues page.
+Feel free to check the [(issues page.)]
 
 
 
-⭐️ Show your support
-Write a message to encourage readers to support your project
+###⭐️ Show your support
+>Write a message to encourage readers to support your project
 
-If you like this project...
+If you like this project you are welcome to contribute!
 
 
 
-🙏 Acknowledgments
-Give credit to everyone who inspired your codebase.
+###🙏 Acknowledgments
+>Give credit to everyone who inspired your codebase.
 
 I would like to thank my coding partner Lethabo who assisted me
 
 
 
-❓ FAQ (OPTIONAL)
-Add at least 2 questions new developers would ask when they decide to use your project.
+###❓ FAQ (OPTIONAL)
+>Add at least 2 questions new developers would ask when they decide to use your project.
+
+1.how to set up linters for html/css. check the link
 
-[how to set up linters for html/css. check the link]
+[(https://www.youtube.com/watch?v=e3I-CIYz1AE&t=4s)]
 
-[https://www.youtube.com/watch?v=e3I-CIYz1AE&t=4s]
-[cloning ssh key in github]
+2.cloning ssh key in github
 
-[https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent]
+[(https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)]
 
 
-📝 License
+###📝 License
 This project is MIT licensed.
 
 NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.
```

Comment:
Well done on making some changes 👍🏽 but as mentioned by the previous reviewer, you should work on formatting your README better using the markdown syntax, you can read about it [here](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax). Some examples are where you use the `#` without a space before the text:
The # without a space:
####Markdown
With a space:
#### Markdown

This makes your README looks very different, I hope this helps 😉